lightning/src/offers/offer.rs

```diff
@@ -1199,6 +1204,9 @@ mod tests {
 			.unwrap();
 		#[cfg(feature = "std")]
 		assert!(!offer.is_expired());
+		let now = std::time::SystemTime::UNIX_EPOCH
+			.elapsed().unwrap();
+		assert!(!offer.is_expired_no_std(now));
```

Comment:
There's another check below for when it is expired. Likewise for the refund test.